.ci/scripts/resolve-dra-manifest.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+ARTIFACT="${ARTIFACT:-$1}"
+BRANCH="${BRANCH:-$2}"
+
+curl -sS https://artifacts-snapshot.elastic.co/$ARTIFACT/latest/$BRANCH.json \
+  | grep build_id \
+  | sed "s/build_id/$ARTIFACT/g" \
```

Comment:
[Jq](https://stedolan.github.io/jq/) is available on our CI agents and much better suited to parsing JSON than `sed`. I suggest we use that.

Reply:
ah I wasn't sure about that and kept it simple. +1

Reply:
done